### ISSUE TITLE
Please support UTF-8

### ISSUE THREAD

**Issue by kseistrup** [robert-lag/brain#2]
I was very surprised to see that `brn` (v2.2) seemingly only supports US-ASCII characters:

```sh
$ brn add -j 'Frække frølår'
error: add_note: the note name 'Frække frølår' contains illegal characters
```

This, essentially, makes Brain useless to me, unless I switch from my native language Danish, to e.g. English (or some other language that uses 7-bit characters only).

Here in year 2022 I feel it's reasonable to expect tool to accept UTF-8.

---

```sh
$ brn --version
Brain 0.1.0  # I bet he says that to all of the girls
$ printf 'LC_ALL=%s, LANG=%s\n' "$LC_ALL" "$LANG"
LC_ALL=en_DK.UTF-8, LANG=en_DK.UTF-8
$ uname -a
Linux mipri 5.18.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 29 Jun 2022 23:03:08 +0000 x86_64 GNU/Linux
```

**Comment by robert-lag** [comment]
I will see what I can do. Shouldn't be too difficult ;)

**Comment by robert-lag** [comment]
Unicode support is now available in version 2.3
